spp_custom_filter_farmer_registry/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "OpenSPP Custom Filter Farmer Registry",
+    "category": "OpenSPP",
+    "version": "17.0.1.0.0",
+    "summary": "Customizes the OpenSPP UI to enhance filtering for Farmer Registry, improving usability and efficiency in managing registrants within social protection programs.",
+    "author": "OpenSPP.org",
+    "website": "https://github.com/OpenSPP/openspp-modules",
+    "license": "LGPL-3",
+    "depends": ["spp_custom_filter_ui", "spp_farmer_registry_base"],
```

Comment:
@shashikala1998 spp_custom_filter_ui is for SPMIS. Please refer to our discussion in Slack. You only need spp_custom_filter in your dependency.